subprojects/gradle/runner/service/src/test/kotlin/com/avito/runner/service/DeviceWorkerTest.kt

```diff
@@ -76,11 +78,12 @@ class DeviceWorkerTest {
                         )
                     )
                 },
-                gettingDeviceStatusResults = intentions.map {
-                    MockActionResult.Success<Device.DeviceStatus>(
-                        Device.DeviceStatus.Alive
+                gettingDeviceStatusResults = listWithDefault(
+                    1 + intentions.size,
```

Comment:
Unfortunately, tests know too much about inner logic.
This is a bad example of custom stubs.
I haven't changed it intentionally to reduce the scope of changes.

We will discuss it additionally in another PR.